modules/hydra-master-common.nix

```diff
@@ -8,7 +8,7 @@ in {
   nix = {
     extraOptions = ''
       auto-optimise-store = true
-      allowed-uris = https://github.com/NixOS/nixpkgs/archive https://github.com/input-output-hk
+      allowed-uris = https://github.com/NixOS/nixpkgs/archive https://github.com/input-output-hk ssh://git@github.com/input-output-hk
```

Comment:
I don't think we should allow ssh by default. We should override where needed. We don't want to accidentally pull something from private repos.

Reply:
This is probably there because someone in another repo needed it though. For some reason the NixOS module doesn't define an option for `allowed-uris` so adding more uris from other modules would require augmenting the module with that option. Shouldn't be hard. Scope it?

Reply:
For now we're just going to `mkForce`-override this where it's needed.